pubnub-gson/pubnub-gson-impl/src/main/java/com/pubnub/internal/java/endpoints/objects_api/memberships/SetMembershipsImpl.java

```diff
@@ -130,4 +132,17 @@ static Collection<? extends com.pubnub.api.models.consumer.objects.PNSortKey<PNM
         }
         return list;
     }
+
+    private String getUserId() {
+        return userId != null ? userId : uuid;
+    }
+
+    private MembershipInclude getMembershipInclude() {
+        if (includeTotalCount || includeCustom || includeType) {
```

Comment:
what about includeChannel? 

maybe flip the if:
if (include != null) // new API is used
else // old API is used

Reply:
Good point. I will fix that.